src/index.ts

```diff
@@ -105,7 +109,15 @@ const weekdaysAlias: { [key: string]: string } = {
   sat: '6'
 }
 
-const hasValidWeekdays = (weekdays: string, alias?: boolean): boolean => {
+const hasValidWeekdays = (weekdays: string, alias?: boolean, allowBlankDay?: boolean): boolean => {
+
+  //
```

Comment:
Oops! I assume you wanted to add a comment about the checks below?

Reply:
Yes, I totally forgot about that 😅